src/cli.go

```diff
@@ -59,7 +60,8 @@ func newAPIClient(ua *userEndpointAuth, cache bool) *sourcegraph.Client {
 
 	var transport http.RoundTripper
 	if cache {
-		transport = http.RoundTripper(httpcache.NewTransport(diskcache.New("/tmp/srclib-cache")))
+		tempdir := filepath.Join(os.TempDir(), "srclib-cache")
```

Comment:
Minor, but shouldn't this local variable be named `tempDir` to follow idiomatic Go style?